drivers/note/Kconfig

```diff
@@ -195,12 +180,9 @@ config DRIVER_NOTECTL
 		If this option is selected, the instrumentation filter control device
 		/dev/notectl is provided.
 
-choice
-	prompt "Note driver selection"
-	default DRIVER_NOTERAM
-
 config DRIVER_NOTERAM
 	bool "Note RAM driver"
+	default y
```

Comment:
let's keep the default off

Reply:
DRIVER_NOTE is off by default
We should enable at least one note driver after enabling DRIVER_NOTE

Reply:
Ok